lib/task/propel/propelBuildNestedSetTask.class.php

```diff
@@ -93,11 +95,6 @@ public function execute($arguments = [], $options = [])
             $this->conn->commit();
         }
 
-        $this->logSection(
-            'propel',
-            'Note: you will need to rebuild your search index for updates to show up properly in search results.'
-        );
-
```

Comment:
It would be nice to have an index flag/option for this task now that it involves ES updates. In some cases it may be nice to index directly like this, but in others it may not be desirable (thinking about maintenance tasks where this is run right before indexing).

I'd set it to false to keep the current behavior by default and change it from the other tasks/jobs calling this one or with a CLI flag.

Reply:
Good call!